src/execution_plan/optimizations/reduce_scans.c

```diff
@@ -52,9 +52,8 @@ static void _reduceScans(ExecutionPlan *plan, OpBase *scan) {
 
 void reduceScans(ExecutionPlan *plan) {
 	// Collect all SCAN operations within the execution plan.
-	OpBase **scans = ExecutionPlan_LocateOps(plan->root,
-											 (OPType_ALL_NODE_SCAN | OPType_NODE_BY_LABEL_SCAN |
-											  OPType_INDEX_SCAN | OPType_NODE_BY_ID_SEEK));
+	const OPType types[] = {OPType_ALL_NODE_SCAN, OPType_NODE_BY_LABEL_SCAN, OPType_INDEX_SCAN, OPType_NODE_BY_ID_SEEK};
```

Comment:
We can introduce a number of static const arrays in `op.h`
e.g.
```
SCAN_OPS = {OPType_ALL_NODE_SCAN, OPType_NODE_BY_LABEL_SCAN, OPType_INDEX_SCAN, OPType_NODE_BY_ID_SEEK};
```

Reply:
I like this idea, and have added static arrays for `PROJECT_OPS` and `TRAVERSE_OPS`. I'm a little leery about `SCAN_OPS`, since callers like this one are introducing `OpType_NODE_BY_LABEL_AND_ID_SCAN` ops, and I don't want to risk either redundant optimizations or misleadingly incomplete arrays.